src/main/java/com/browserstack/automate/ci/jenkins/BrowserStackCredentials.java

```diff
@@ -127,7 +128,7 @@ public static FormValidation testAuthentication(final String username, final Str
             return FormValidation.error(e.getMessage());
         }
 
-        return FormValidation.error(ERR_INVALID_AUTH);
+        return FormValidation.error(ERR_HUB_UNREACHABLE);
```

Comment:
when response code is [403/408,](https://github.com/jenkinsci/browserstack-integration-plugin/pull/19/files#diff-df6b62a67f35c2a68cdee332bb91ad8cR124) this needs to return correct message and not "unable to reach BrowserStack"

Reply:
`e.getMessage()` will now be displayed for errors other than 401s